### PR TITLE
[Issue 45]: Pass HTTP request headers to cells

### PR DESCRIPTION
**Pull request by Lull3rSkat3r**
Implementation of #45.
(c) Copyright IBM Corp. 2015

### CODE REVIEW

**Comment by Lull3rSkat3r** [comment]
Rebased.

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ The REQUEST object currently contains the following properties:
 * `args` - An object with keys representing query parameter names and their associated values. A query parameter name may be specified multiple times in a valid URL, and so each value is a sequence (e.g., list, array) of strings from the original URL.
 * `path` - An object of key-value pairs representing path parameters and
 their values.
+* `headers` - A JSON object representing the HTTP headers of the request
```
Comment (parente) [inline README.md]: JSON object? :)

Comment (Lull3rSkat3r) [inline README.md]: :D Today is not my day. Just pushed a change.

File: README.md [hunk 2]
```diff
@@ -210,6 +210,7 @@ The REQUEST object currently contains the following properties:
 * `args` - An object with keys representing query parameter names and their associated values. A query parameter name may be specified multiple times in a valid URL, and so each value is a sequence (e.g., list, array) of strings from the original URL.
 * `path` - An object of key-value pairs representing path parameters and
 their values.
+* `headers` - An object of key-value pairs object representing the HTTP headers of the request
```
Comment (parente) [inline README.md]: One more typo. Object ... object.

**Pull request merged**